fdbserver/workloads/ConfigureDatabase.actor.cpp

```diff
@@ -273,6 +273,14 @@ struct ConfigureDatabaseWorkload : TestWorkload {
 
 	ACTOR Future<Void> _setup(Database cx, ConfigureDatabaseWorkload* self) {
 		wait(success(ManagementAPI::changeConfig(cx.getReference(), "single storage_migration_type=aggressive", true)));
+
+		// Redwood is the only storage engine type supporting encryption.
+		DatabaseConfiguration config = wait(getDatabaseConfiguration(cx));
+		if (config.encryptionAtRestMode.isEncryptionEnabled()) {
+			self->storageEngineExcludeTypes = { 0, 1, 2, 4, 5 };
```

Comment:
Would it be more helpful to store these as strings instead of numbers?

Reply:
With the test spec we used to specify `storageEngineExcludeTypes` with numbers and I want to keep it consistent. For each storage engine type, they could have alias, so it is probably easier to just store them as numbers.